src/components/AutoComplete/AutoComplete.js

```diff
@@ -60,12 +60,21 @@ export default class AutoComplete extends Component {
 
   setPermalinkParams = (param, value) => {
     const permalink = new URLSearchParams(window.location.search);
-    const paramItem = this.props.source.filter(item => item.label === value);
-    if (paramItem.length && value.length) {
-      permalink.set(param, paramItem[0].value);
-    } else if (!value.length) {
-      permalink.delete(param);
+
+    switch (param) {
```

Comment:
Why?

Reply:
That's a mistake on my part, sorry. I misunderstood that code (probably shouldn't do this in the evening when tired). Will fix both things. Thank you for the review!